.gitignore

```diff
@@ -75,7 +75,12 @@ node_modules
 # Webstorm development environment
 .idea/
 
+# Vim swap files
+*.swp
+*.swo
```

Comment:
ONe note you can add globally those files to global gitignore you don't need to commit that to the repo when you use vim:

See SO: [Global Git ignore](https://stackoverflow.com/q/7335420/387194)